### PR TITLE
modal tests with enzyme

### PR DESCRIPTION
**Pull request by rgarcia**
This PR uncomments the modal tests and ports them to use enzyme

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,8 @@
     "eslint-plugin-react": "^4.2.3",
     "file-loader": "^0.8.5",
     "ignore-styles": "^1.2.0",
+    "jsdom": "^8.4.0",
+    "jsdom-global": "^1.7.0",
```
Comment (rgarcia) [inline package.json]: had to add these deps (plus `--require jsdom-global/register`) to get the Modal code that operates on `window` to run in tests

File: test/Modal_test.jsx [hunk 2]
```diff
@@ -1,54 +1,39 @@
...
+  it("renders the background, modal window, title, and contents", () => {
+    const modal = shallow(
+      <Modal title="My Title">
+        <div>Example Content</div>
+      </Modal>
+    );
+    const background = modal.find(".Modal--background");
+    const windoe = modal.find(".Modal--window");
+    const header = modal.find(".Modal--header");
+    const content = modal.find(".Modal--window--content");
```
Comment (jacobsimon) [inline test/Modal_test.jsx]: you can do `find("div.Modal--background")` and then you don't need the type assertions

Comment (rgarcia) [inline test/Modal_test.jsx]: 420b2c0

File: test/Modal_test.jsx [hunk 3]
```diff
@@ -1,54 +1,35 @@
...
+    const header = modal.find("div.Modal--header");
+    const content = modal.find("div.Modal--window--content");
+    assert.equal(background.length, 1);
+    assert.equal(windoe.length, 1);
+    assert.deepEqual(windoe.prop("style"), {marginLeft: "-200px", width: "400px"});
+    assert.equal(header.length, 1);
+    assert.equal(header.find("h2").length, 1);
+    assert.equal(header.find("h2").text(), "My Title");
+    assert.equal(content.length, 1);
+    assert.equal(content.text(), "Example Content");
```
Comment (kshen0) [inline test/Modal_test.jsx]: Worth breaking these into separate tests?

Comment (rgarcia) [inline test/Modal_test.jsx]: going to leave it as is for now, since I'd like to keep this PR limited to porting the existing structure of the tests

**Pull request merged**